src/core/service.c

```diff
@@ -1263,10 +1262,6 @@ static int service_spawn(
         if (r < 0)
                 return r;
 
-        r = unit_full_printf_strv(UNIT(s), c->argv, &argv);
-        if (r < 0)
```

Comment:
So, I thought about making it work so that we resolve with the Slice= setting in effect so far. This means the cgroup path would use the default "system.slice" for any %c occasions before the first Slice= and from that point on use the configured once. But that is awful too, as in the above example the Description= would almost certainly resolve the wrong path then, as the whole [Unit] section is placed before the [Service] section usually, and that's where Slice= is usually placed...

What a fuckup. And all my fault.. Grmbl.

I am tempted to say that we have to deprecate %c altogether. Specifiers that depend on the configuration of its own unit file can never work...